docs/hadoop-catalog.md

```diff
@@ -25,19 +25,88 @@ Hadoop 3. If there's any compatibility issue, please create an [issue](https://g
+| `authentication.kerberos.keytab-uri`               | The URI of The keytab for the Kerberos authentication.                                                                                                                                                                                                                                                        | (none)          | required if the value of `authentication.type` is Kerberos. | 0.5.1            |
+| `authentication.kerberos.check-interval-sec`       | The check interval of Kerberos credential for Hadoop catalog.                                                                                                                                                                                                                                                 | 60              | No                                                          | 0.5.1            |
+| `authentication.kerberos.keytab-fetch-timeout-sec` | The fetch timeout of retrieving Kerberos keytab from `authentication.kerberos.keytab-uri`.                                                                                                                                                                                                                    | 60              | No                                                          | 0.5.1            |
+
+#### S3 fileset
+
+| Configuration item             | Description                                                                                                                                                                           | Default value   | Required                   | Since version    |
+|--------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|----------------------------|------------------|
+| `filesystem-providers`         | The file system providers to add. Set it to `s3` if it's a s3 fileset or a comma separated string that contains `s3` like `gs,s3` to support multiple kind of fileset including `s3`. | (none)          | Yes                        | 0.7.0-incubating |
+| `default-filesystem-provider`  | The name default filesystem providers of this Hadoop catalog if users do not specify the scheme in the URI. Default value is `builtin-local`, for S3, if we set this value, we can omit the prefix 'oss://' in the location. | `builtin-local` | No                         | 0.7.0-incubating |
```

Comment:
Why you mentioned S3 here, but the prefix is "oss://"?

Reply:
fixed